src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -506,6 +506,35 @@ public void delete() throws IOException {
     }
 
     /**
+     * Sort orders for listing forks
+     */
+    public static enum Sort { NEWEST, OLDEST, STARGAZERS }
+
+    /**
+     * Lists all the forks of this repository.
+     */
+    public PagedIterable<GHRepository> listForks() {
+      return listForks(null);
+    }
+
+    /**
+     * Lists up all the forks of this repository, sorted by the given sort order.
+     */
+    public PagedIterable<GHRepository> listForks(final Sort sort) {
```

Comment:
Agreed.

Just added (hopefully) better Javadoc. Unfortunately, I was not able to find out how to add the `@CheckForNull` annotation.